kong/templates/kong_defaults.lua

```diff
@@ -25,6 +25,7 @@ cluster_cert_key = NONE
 cluster_mtls = shared
 cluster_ca_cert = NONE
 cluster_server_name = NONE
+cluster_status_ttl = 1209600
```

Comment:
Thinking form a user who doesn't know about internals of Kong, `ttl` in this sense is a pretty confusing acronym to pick for this configuration property. The fact that internally it is driven by `ttl` mechanism is an implementation detail that shouldn't surface here.
Would renaming this to something along the lines `cluster_node_purge_delay` be an improvement?